vagrant/provision/once-as-vagrant.sh

```diff
@@ -44,3 +44,14 @@ echo "Done!"
 info "Enabling colorized prompt for guest console"
 sed -i "s/#force_color_prompt=yes/force_color_prompt=yes/" /home/vagrant/.bashrc
 echo "Done!"
+
+info "Enabling access to GII from 192.168.83.1"
+sed  -i  '/return $config/d' ./frontend/config/main-local.php
+cat <<EOT >> ./frontend/config/main-local.php
+if (!YII_ENV_TEST && isset(\$config['modules']['gii']) )
+    \$config['modules']['gii']['allowedIPs'] = ['127.0.0.1', '::1','192.168.83.1'];
+
```

Comment:
Configurable only host ip and netmask but not guest ip, it is allways netmask with `1` on the end. But i will try to make it based on ip from config